pkg/doc/verifiable/presentation.go

```diff
@@ -189,7 +194,7 @@ func (vp *Presentation) Credentials() []interface{} {
 func (vp *Presentation) SetCredentials(creds ...interface{}) error {
```

Comment:
I didn't notice this method from an earlier PR - why do credentials need to be set after instantiation?

Reply:
It's related to the scenario when a Holder wants to prepare a Verifiable Presentation to send to a Verifier. The Presentation can contain one or more Verifiable Credentials. They could be stored in the Holder's wallet. `SetCredentials()` allows attaching the Credentials to the Presentation.